### PR TITLE
1. draft on limitations

### PR DESCRIPTION
**Pull request by AnonymousPoster**
This is my first draft for review and comments

Related #76

Please don't hesitate to give any kind of feedback's, all helps improve this current draft

### CODE REVIEW

File: docs/components/databases.rst [hunk 1]
```diff
@@ -104,3 +104,23 @@ How to use the :code:`mysql` or :code:`mariadb` format?
...
+Known limitations with MySQL and MariaDB
+----------------------------------------
+
+Do to a default setting that prevent users from creating stored functions that 
+will cause unsafe events to be written into the binary log, you will need to 
+contact your database manger (maybe your self) and ask them to
+
+1. set :code:`set global log_bin_trust_function_creators=1;` for your database **or**
+
+2. Import the given SQL script your self as SUPER-PRIVILIDGED user such as root.
```
Comment (spirillen) [inline docs/components/databases.rst]: @AnonymousPoster I'll suggest a minor rewrite...

s/Import the given SQL script your self/Import the matching SQL script your self/

File: docs/components/databases.rst [hunk 2]
```diff
@@ -104,3 +104,23 @@ How to use the :code:`mysql` or :code:`mariadb` format?
 .. note::
     If the environment variables are not found, you will be asked to prompt the 
     information.
+
+Known limitations with MySQL and MariaDB
+----------------------------------------
+
+Do to a default setting that prevent users from creating stored functions that 
+will cause unsafe events to be written into the binary log, you will need to 
+contact your database manger (maybe your self) and ask them to
```
Comment (spirillen) [inline docs/components/databases.rst]: It a little heavy to read. and to much back and forth. I suggestion a second rewrite, to soften the reading and enhancing the understanding.

Good work @AnonymousPoster

Comment (AnonymousPoster) [reply docs/components/databases.rst]: I agree. I'll take a second review and see what I can do :smiling_imp:

Comment (AnonymousPoster) [reply docs/components/databases.rst]: I've some minor changes that at first eyesight looks ~ok

https://github.com/funilrys/PyFunceble/pull/81

**Pull request merged**